cmd/kubeapps-apis/proto/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2.proto

```diff
@@ -7,10 +7,16 @@ import "kubeappsapis/core/packages/v1alpha1/packages.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 service FluxV2PackagesService {
-  // GetAvailablePackages returns the available packages managed by the 'fluxv2' plugin
-  rpc GetAvailablePackages(kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesResponse) {
+  // GetAvailablePackageSummaries returns the available packages managed by the 'fluxv2' plugin
+  rpc GetAvailablePackageSummaries(kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesResponse) {
     option (google.api.http) = {
-      get: "/plugins/fluxv2/packages/v1alpha1/packages"
+      get: "/plugins/fluxv2/packages/v1alpha1/availablepackagesummaries"
```

Comment:
Sorry, yes I had a TODO to add http endpoints to the doc but hadn't yet addressed it yet. They're easy to change though for now (while we don't have a client :) ). These are fine for now.